packages/core/src/components/DataTable/columns/ExpandableColumn.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Column } from 'react-virtualized';
+import IconChevronDown from '@airbnb/lunar-icons/lib/interface/IconChevronDown';
+import IconChevronRight from '@airbnb/lunar-icons/lib/interface/IconChevronRight';
+import { css, WithStylesProps } from '../../../composers/withStyles';
+import Spacing from '../../Spacing';
+import { TableRow } from '../types';
+import { EXPANDABLE_COLUMN_WIDTH } from '../constants';
+
+export default function ExpandableColumn(
```

Comment:
Is this a component or function?

Reply:
All these functions that act like pseudo components, lets prefix them with `render`, so `renderExpandableColumn`.